busmall.js

```diff
@@ -110,62 +108,66 @@ function displayTable() {
     ul.appendChild(li);
 
   }
-  section.appendChild(ul);
+  sectionTotal.setAttribute('style', 'border: 10px double black');
+  sectionTotal.appendChild(h3);
+  sectionTotal.appendChild(ul);
 }
 
-/* 
 function renderChart() {
   var ctx = document.getElementById('chart').getContext('2d');
   var chart = new Chart(ctx, {
```

Comment:
Since you never use this `Chart` after you have created it, you do not need to store it in a variable.